pubspec.yaml

```diff
@@ -189,7 +189,7 @@ dependencies:
   # flutter_localizations depends on intl 0.18.0
   intl: 0.18.0
 
-  super_tag_editor: 0.1.3
+  super_tag_editor: 0.1.4
```

Comment:
Needs this https://github.com/dab246/super_tag_editor/pull/21 to be merged and published first